### PR TITLE
[fuchsia_ctl] try to fix tree by adding dart version annotations

### PR DESCRIPTION
**Pull request by gaaclarke**

### CODE REVIEW

**Comment by gaaclarke** [comment]
Landing on red now to fix the tree @goderbauer please review when you get a chance.

File: packages/fuchsia_ctl/test/command_line_test.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
+// @dart = 2.4
```
Comment (goderbauer) [inline packages/fuchsia_ctl/test/command_line_test.dart:4]: Do you actually still need these opt-outs? I would have expected that upgrading package:test is enough...

**Review by goderbauer — COMMENTED** [review]
LGTM

File: packages/fuchsia_ctl/test/command_line_test.dart (hunk 1 above)
Comment (gaaclarke) [reply packages/fuchsia_ctl/test/command_line_test.dart:4]: It was the first thing I tried, it didn't fix the issues without the upgrade.  I just left them in without trying to remove it.

**Pull request merged**